src/templates/static/multi_account_manager.html.tt

```diff
@@ -0,0 +1,123 @@
+                    </div>
+                </div>
+                <div class="col-md-8 push-md-2 col-lg-4 push-lg-0">
+                    <div class="box fill-bg-color">
+                        <span class="fx fx-trade-allocations"></span>
+                        <p>[% l('Clients\' trade allocations start from 0.01 lots') %]</p>
+                    </div>
+                </div>
+                <div class="col-md-8 push-md-2 col-lg-4 push-lg-0">
+                    <div class="box fill-bg-color">
+                        <span class="fx fx-allow-expert"></span>
+                        <p>[% l('Clients can close out trades') %]</p>
+                    </div>
+                </div>
+            </div>
+        </div>
+
+        <div class="separator fill-bg-color"></div>
+
+        <div class="row center-text">
+            <div class="col-sm-12">
+                <h2>[% l('How it works') %]</h2>
+            </div>
+            <div class="col-md-8 push-md-2 col-lg-4 push-lg-0">
+                <div class="box">
+                    <span class="fx fx-talk"></span>
+                    <h4>[% l('Talk to us') %]</h4>
+                    <p>[% l('Learn how to set up and authenticate your master account, link sub accounts, and more.') %]</p>
+                </div>
+            </div>
+            <div class="col-md-8 push-md-2 col-lg-4 push-lg-0">
+                <div class="box">
+                    <span class="fx fx-download"></span>
+                    <h4>[% l('Download MAM') %]</h4>
+                    <p>[% l('Download the MAM application after setup is completed. Log in with your master account credentials.') %]</p>
+                </div>
+                <div class="fx-arrow-right hidden-md-down">
+                    <span class="fx-arrow-right-icon"></span>
+                </div>
+            </div>
+            <div class="col-md-8 push-md-2 col-lg-4 push-lg-0">
+                <div class="box">
+                    <span class="fx fx-manage-subaccounts"></span>
+                    <h4>[% l('Manage sub accounts') %]</h4>
+                    <p>[% l('Monitor and manage your client list, set allocations and exclusions for individual sub accounts, and more.') %]</p>
+                </div>
+                <div class="fx-arrow-right hidden-md-down">
+                    <span class="fx-arrow-right-icon"></span>
+                </div>
+            </div>
+            <div class="col-sm-12">
+                <p class="center-text">
+                    <a class="button" href="#">
+                        <span>[% l('Download now') %]</span>
+                    </a>
+                </p>
+            </div>
+            <div class="col-md-10 push-md-1">
+                <div class="box center-text fill-bg-color">
+                    <span>[% l('Interested in our MAM tool for MetaTrader 5? Contact us at marketing@binary.com for more info.') %]</span>
```

Comment:
The correct email address is: `marketing@champion-fx.com` and please make it a link.